Formula/xcbuild.rb

```diff
@@ -0,0 +1,24 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git",
+    :tag => "0.1.0",
+    :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa",
+    :shallow => false
+  head "https://github.com/facebook/xcbuild.git",
+    :shallow => false
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
```

Comment:
I read from the above comment that you need/would likely want `ninja` at runtime as well as buildtime?

We have a similar syntax to Apt here. Just:
```ruby
depends_on "ninja" => :recommended
```